### PR TITLE
Bullet spacing fix + 2 new git resources

### PR DESCRIPTION
**Pull request by crumdev**

### CODE REVIEW

**Comment by crumdev** [comment]
There were multiple ```<html>, <body>, and <!doctype html>``` tags throught the document. Just need to have one of each i.e.:
```html5
<!doctype html>
<html lang="en">
<head>Load CSS</head>
<body>
  All content here
  <script work best loaded before the closing body tag>
</body>
</html>
```

File: index.html [hunk 1]
```diff
@@ -1,249 +1,339 @@
...
+  <!--      // 5. The API calls this function when the player's state changes.-->
+  <!--      //    The function indicates that when playing a video (state=1),-->
+  <!--      //    the player should play for six seconds and then stop.-->
+  <!--      var done = false;-->
+  <!--      function onPlayerStateChange(event) {-->
+  <!--         // if (event.data == YT.PlayerState.PLAYING && !done) {-->
+  <!--         //   setTimeout(stopVideo, 6000);-->
+  <!--         //   done = true;-->
+  <!--         // }-->
+  <!--     //   if (event.data == YT.PlayerState.ENDED) {-->
```
Comment (mjgila1c) [inline index.html]: What I'm trying to achieve with this YouTube API is to have the video Fade Out once the PlayerState reaches the ENDED point. I don't think I need all the HTML code I added, but I can't seem to figure out how to have the video Fade Out. Do you have any suggests?

**Review by mjgila1c — APPROVED** [review]
crumdev, I really appreciate your help on this. As this is my first repo Pull Request I hope I'm doing this right. I did have one question on how to handle the YT API, I've added a comment in  the code if you'd like to respond or offer suggestion.

**Comment by crumdev** [comment]
It looks like you merged it just fine. As for the youtube api I would remove all the code you have and start by just having the one "div" with id=player and then copy everything they have [here](https://developers.google.com/youtube/iframe_api_reference#Getting_Started) in between the <script> tags into your external app.js file so it is easier to read and tweak. 

The parts you will want to change are the videoID to the one you want to play then look at the function with the comment marked "5." It calls that function whenever the state of the player changes (i.e. queued, playing, **ended**). I'll let you try it first. the function currently checks to see if the player state is "playing" and that their on variable done has not been set to true yet then it sets a 6 second time out, stops the video, and sets done to true.

Check here for the events you can check for [YoutubeAPI Events](https://developers.google.com/youtube/iframe_api_reference#Events)

Once you found your event use the if statement to see if your video has ended. Then add some javascript to fade your player out. I see you have jquery added to the site. Try googling Jquery FadeOut

**Pull request merged**